src/sentry/api/helpers/group_index/delete.py

```diff
@@ -53,6 +54,25 @@ def delete_group_list(
         if g.issue_category == GroupCategory.ERROR:
             error_ids.append(g.id)
 
+    transaction_id = uuid4().hex
+    delete_logger.info(
+        "object.delete.api",
+        extra={
+            "objects": group_ids,
+            "project_id": project.id,
+            "organization_slug": project.organization.slug,
+            "transaction_id": transaction_id,
```

Comment:
If we find in GCP the `transaction_id`, we can then use it in Sentry to determine if there are errors that happened either in the API call or one of the tasks.